src/connection/mod.rs

```diff
@@ -41,7 +41,7 @@ pub fn authenticate(transport: Transport, credentials: Credentials, device_id: S
         system_info => {
             cpu_family: CpuFamily::CPU_UNKNOWN,
             os: Os::OS_UNKNOWN,
-            system_information_string: "This is not the client you are looking for".to_owned(),
+            system_information_string: "librespot".to_string() + "_" + &(version::short_sha()) + "_" + &(version::short_now()),
```

Comment:
You can use `format!("librespot-{}-{}", version::short_sha(), version::short_now())`